react-components/src/utilities/fdmViewsExist.ts

```diff
@@ -0,0 +1,12 @@
+/*!
+ * Copyright 2024 Cognite AS
+ */
+import { type FdmSDK, type Source } from './FdmSDK';
+
+export async function fdmViewsExist(fdmSdk: FdmSDK, neededViews: Source[]): Promise<boolean> {
+  const views = await fdmSdk.getViewsByIds(neededViews);
+  if (views.items.length === neededViews.length) {
+    return true;
+  }
+  return false;
+}
```

Comment:
I know that it is not part of this refactoring and fixes but I think we should consider to not just returning boolean if some of the views doesnt exist by calling this method. We should also inform which one doesnt exist if the request is for multiple views.